tests/test_messages_config/test_messages_config.py

```diff
@@ -2,40 +2,85 @@
 import sys
 
 
-def test_modified_message():
-    """Test the presence of modified custom message."""
+def test_valid_message_config_and_pyta_overwrite():
+    """Test the error messages given a valid messages-config-path with use-pyta-error-messages as True"""
     output = subprocess.run(
         [
             sys.executable,
             "-m",
             "python_ta",
             "--config",
-            "tests/test_messages_config/test.pylintrc",
-            "examples/pylint/e0111_bad_reversed_sequence.py",
+            "tests/test_messages_config/test1.pylintrc",
+            "tests/test_messages_config/testing_code.py",
         ],
         capture_output=True,
         text=True,
     )
 
-    assert "This custom error message is modified." in output.stdout
+    assert (
+        "This custom error message is modified." in output.stdout
+        and "The first reversed() argument is not a sequence" not in output.stdout
```

Comment:
In general do not combine boolean expressions in the same assert statement. It's okay here to use two different assert statements; this will give you more information if an assertion fails.